pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[build-system]
+requires = ["flit"]
+build-backend = "flit.buildapi"
+
+[tool.flit.metadata]
+module = "braubuddy"
+author = "James Stewart"
```

Comment:
Is it legit to put 2 x names here? I feel a bit dodgy being the only name on this if we're building it together...